cmd/export.go

```diff
@@ -0,0 +1,123 @@
+// ------------------------------------------------------------------------
+// SPDX-FileCopyrightText: Copyright © 2024 bomctl authors
+// SPDX-FileName: cmd/export.go
+// SPDX-FileType: SOURCE
+// SPDX-License-Identifier: Apache-2.0
+// ------------------------------------------------------------------------
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+package cmd
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/bomctl/bomctl/internal/pkg/db"
+	"github.com/bomctl/bomctl/internal/pkg/export"
+	"github.com/bomctl/bomctl/internal/pkg/utils"
+	"github.com/bomctl/bomctl/internal/pkg/utils/format"
+)
+
+func exportCmd() *cobra.Command {
+	documentIDs := []string{}
+	opts := &export.ExportOptions{
+		Logger: utils.NewLogger("export"),
+	}
```

Comment:
Since the logger is instantiated inside `initBackend` right?

```suggestion
	opts := &export.ExportOptions{}
```